tensilelite/Tensile/Components/LSU.py

```diff
@@ -245,6 +242,12 @@ def writeReadReduction(self, writer, kernel):
                 DSLoadBX     = DSLoadB32
                 numInstPerVW = bytesPerVector // 4
                 regsPerStore = 1
+            maxOffset =  kernel["LocalSplitU"] * ldsStride + ((numVgprPerLSU // self.LSUfullVw) * numInstPerVW + numInstPerVW) * regsPerStore * (bpr * kernel["WavefrontSize"])
```

Comment:
I think this is not accurate.
This should be
kernel["LocalSplitU"] -> kernel["LocalSplitU"] - 1
(numVgprPerLSU // self.LSUfullVw) -> (numVgprPerLSU // self.LSUfullVw) - 1
numInstPerVW -> numInstPerVW - 1

Would you please make sure this matches the actual maximum values?

Reply:
I think your change is still inaccurate.

This is the code to calculate the offset
                        regIdx = (i * numInstPerVW + v) * regsPerStore
                        offset = r * ldsStride + regIdx * (bpr * kernel["WavefrontSize"])

You should replace "v" with numInstPerVW - 1.
Would you add some debug code (print or compare with max value you calculate) and make sure the max value is accurate?